lib/public/services/detectors/detectorsProvider.js

```diff
@@ -43,14 +52,54 @@ export class DetectorsProvider extends Observable {
     }
 
     /**
-     * Returns the list of all the physical available detectors
-     * E.g `TST` detector is not phsycial system so it is no included in the results
+     * Return all detectors list observable data
+     *
+     * @return {ObservableData<RemoteData<Detector[], ApiError>>} the observable detectors list
+     */
+    get all$() {
+        if (this._isStale()) {
+            this._load();
+        }
+        return this._all$;
+    }
+
+    /**
+     * Return physical (meaning actual detectors, for example excluding TST) detectors list observable data
+     *
```

Comment:
I would remove `for example`, as it's precisely: `excluding TST` and nothing more for now

Reply:
ok, maybe, as I see my previous docs had same notion

Reply:
I think it's better to keep as an example to hide the actual implementation, maybe at some point we'll have other tests detectors? As you like, I was simply rephrasing your comment